README.md

```diff
@@ -44,16 +43,16 @@ installation options for a variety of operating systems:
 ### R-universe
 
 [R-universe](https://rpolars.r-universe.dev/polars#install) provides
-pre-compiled **polars** binaries for Windows and MacOS (x86_64), with
+pre-compiled **polars** binaries for Windows and MacOS (x86\_64), with
 source builds for other platforms. Please see further below for binary
 install options on Linux.
 
 ``` r
 install.packages("polars", repos = "https://rpolars.r-universe.dev")
 ```
 
-Special thanks to Jeroen Ooms ([@jeroen](https://github.com/jeroen)) for
-the excellent R-universe support.
+Special thanks to Jeroen Ooms (\[@jeroen\](<https://github.com/jeroen>))
```

Comment:
ditto.

Reply:
My knitr behaves very weirdly and escapes these links automatically. However, I have not modified the original links in .Rmd and they look fine. So I just edited the .md files in https://github.com/pola-rs/r-polars/commit/d39920679949d80655934c2225b1321879004e5e